README.md

```diff
@@ -366,7 +366,7 @@ function mapStateToProps(state) {
 }
 
 function mapDispatchToProps(dispatch) {
-  return bindActionCreators(Object.assign({}, todoActionCreators, counterActionCreators), dispatch);
+  return bindActionCreators({ ...todoActionCreators, ...counterActionCreators}, dispatch);
```

Comment:
There are several other instances of object spread notation in the README, both in the text and in the examples. If you want, I can revert them all back to Object.assign.